rust/noosphere/src/ffi/authority.rs

```diff
@@ -0,0 +1,125 @@
+use std::ffi::c_void;
+
+use cid::Cid;
+use noosphere_core::{
+    authority::Authorization,
+    data::{Did, Mnemonic},
+};
+use noosphere_sphere::{HasMutableSphereContext, SphereAuthorityEscalate, SphereAuthorityWrite};
+use safer_ffi::prelude::*;
+
+use crate::{
+    error::NoosphereError,
+    ffi::{NsError, NsNoosphere, NsSphere},
+};
+
+#[ffi_export]
+/// @memberof ns_sphere_t
+///
+/// Authorize another key to manipulate this sphere, given a display name for
+/// the authorization and the DID of the key to be authorized.
```

Comment:
consider adding `public` e.g. `"given a public display name for the authorization"`